src/platforms/common/index.mdx

```diff
@@ -34,6 +34,7 @@ This snippet includes an intentional error, so you can test that everything is w
 
 <Note>
 
+Keep in mind that errors triggered within Browser DevTools are sandboxed and they will not trigger an error handler. Place the snippet directly in your code instead.
```

Comment:
This will appear on the Getting Stated page for every SDK - which I suspect we don't want as this is a note specific to JS. 

I'd recommend instead putting this in the `includes/getting-started-verify/javascript.mdx` file.

```suggestion
Errors triggered from within Browser DevTools are sandboxed, so will not trigger an error handler. Place the snippet directly in your code instead.
```